Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMHTMLFrameElement.cpp

```diff
@@ -184,12 +184,6 @@ static void webkit_dom_html_frame_element_get_property(GObject* object, guint pr
     case DOM_HTML_FRAME_ELEMENT_PROP_CONTENT_WINDOW:
         g_value_set_object(value, webkit_dom_html_frame_element_get_content_window(self));
         break;
-    case DOM_HTML_FRAME_ELEMENT_PROP_WIDTH:
-        g_value_set_long(value, webkit_dom_html_frame_element_get_width(self));
-        break;
-    case DOM_HTML_FRAME_ELEMENT_PROP_HEIGHT:
-        g_value_set_long(value, webkit_dom_html_frame_element_get_height(self));
-        break;
```

Comment:
For example, here you would just remove the calls to g_value_set_long(), but not remove the cases.

Reply:
Done.